packages/react-core/src/demos/DataListDemo.md

```diff
@@ -28,3 +29,8 @@ import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/Dashboard
 
 ```js file="./DataList/examples/DataListExpandableControlInToolbar.tsx" isFullscreen
 ```
+
+### Pagination
```

Comment:
If this is to create a static bottom pagination demo, I'd rename this. The core demo is called "Static bottom pagination"